app/presenters/two_factor_auth_code/totpable.rb

```diff
@@ -0,0 +1,20 @@
+module TwoFactorAuthCode
+  module Totpable
+    def auth_app_fallback_link
+      return empty unless totp_enabled
+
+      t('links.phone_confirmation.auth_app_fallback_html', link: auth_app_fallback_tag)
+    end
```

Comment:
I'm not sure why Code Climate thinks this is not tested, but I verified that it is.